AppSyncRealTimeClient/ConnectionProvider/AppsyncRealtimeConnection/RealtimeConnectionProvider+Websocket.swift

```diff
@@ -103,15 +113,15 @@ extension RealtimeConnectionProvider: AppSyncWebsocketDelegate {
                 updateCallback(event: .data(appSyncResponse))
             }
         case .keepAlive:
-            print("")
+            break
```

Comment:
should we replace this with verbose logging instead? like "Received keep alive message on subscription connection"

Reply:
Good call, will do